include/zephyr/bluetooth/conn.h

```diff
@@ -640,6 +640,10 @@ struct bt_conn_le_cs_subevent_result {
 		/** Number of CS steps in the subevent.
 		 */
 		uint8_t num_steps_reported;
+		/** Step number, on which the subevent was aborted
```

Comment:
we should also add some docs here https://github.com/zephyrproject-rtos/zephyr/pull/79497/files#diff-4538e90f4e61dd34802aa1c0a5382dc89d4df6032ec6a3dbbf4eef746ebdd488R622 about how abort_step relates to the status

For instance I expect that if 200 steps are reported like this:
- subevent result: 100 steps, subevent_done_status = 0x1
- subevent result continue: 99 steps, subevent_done_status = 0x1
- subevent result continue: 1 step, subevent_done_status = 0xF

this struct will have subevent_done_status = 0xF and step_data_buf pointing to 200 step data elements.
So we should clarify in the API that the user should interpret this as "199 non-aborted steps + 1 failed/aborted step".